lib/beacon/live_admin/page_live.ex

```diff
@@ -26,7 +26,19 @@ defmodule Beacon.LiveAdmin.PageLive do
     Cluster.maybe_discover_sites()
 
     sites = Beacon.LiveAdmin.Cluster.running_sites()
-    %{"pages" => pages, "beacon_live_admin_page_url" => current_url} = session
```

Comment:
Raising with a custom message is more explicit and there's no need to try/rescue either. You can do something like:

```
current_url = Map.get(session, "beacon_live_admin_page_url") || raise """
failed to resolve page URLs

You must add Beacon.LiveAdmin.Plug to your :browser pipeline that beacon_live_admin is piped through
"""
```

Reply:
Much more simple than my convoluted solution.  Thanks.  I've updated the PR.